app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -198,6 +200,8 @@ def load_and_authorize_plan
 
       def load_dynflow_plan(plan_id)
         Rails.application.dynflow.world.persistence.load_execution_plan(plan_id)
+      rescue
```

Comment:
should we at least log the error in debug mode? there's an exception logger I think

Reply:
:dart: that is what I was looking for, thanks :+1: